images/build/Dockerfiles/gitHubActions.Dockerfile

```diff
@@ -66,6 +66,7 @@ RUN if [ "${DEBIAN_FLAVOR}" = "bookworm" ]; then \
             libssl3 \
             libyaml-dev \
             libxml2 \
+            # https://github.com/yaml/pyyaml/issues/724#issuecomment-1638591821
```

Comment:
Do we need this comment? It seems out of place

Reply:
Removing this in the next commit. Forgot to delete this...